client/bindings/java/iota-client-java/CHANGELOG.md

```diff
@@ -19,6 +19,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.0-rc.2 - 2022-10-17
```

Comment:
Just checking, is there no other file you also need to change this ?

Reply:
Hm not sure I understand, what do you mean?

Reply:
Don't you need to bump the version somewhere if you want to release it ?

Reply:
Oh, right. Forgot about the build.gradle